app/liquid_tags/youtube_tag.rb

```diff
@@ -1,6 +1,8 @@
 class YoutubeTag < LiquidTagBase
   PARTIAL = "liquids/youtube".freeze
-  REGISTRY_REGEXP = %r{https?://(www\.)?youtube\.(com|be)/(embed|watch)?(\?v=)?(/)?[a-zA-Z0-9_-]{11}((\?t=)?(\d{1,})?)?}
+  # rubocop:disable Layout/LineLength
+  REGISTRY_REGEXP = %r{https?://(www\.)?(youtube\.com|youtu\.be)?/(embed|watch)?(\?v=)?(/)?[a-zA-Z0-9_-]{11}((\?t=)?(\d{1,})?)?}
```

Comment:
nitpick: adding the time to youtube urls seems like it's not related to the vimeo embed changes, this could have been a second PR (or included separate instructions to test the change with a YT link including time)?

Reply:
Hey @djuber, actually, the change here was in the domain regex: `youtube\.(com | be)` vs `(youtube\.com | youtu\.be)` not the time; the testing of the time parameter in the youtube URL was done in [this PR](https://github.com/forem/forem/pull/15420).

Reply:
yep - re-reading the change I saw what happened - makes sense.